python/tests/utils/test_datetime.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime, tzinfo
```

Comment:
Can you add the Apache header here? Otherwise the tests will fail

Reply:
Ah, okay. Added it